### PR TITLE
feat: Add interaction button to table cell

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description
Users need be able to add an interaction to a company directly from the homepage. I have added an "add interaction" button against each company listed in the company list table, each button will send the user to the "add interaction" flow related to the company.

## Screenshot
Screenshot 2019-11-22 at 17 18 04

### CODE REVIEW

File: src/dashboard/my-companies/MyCompaniesTable.jsx [hunk 1]
```diff
@@ -21,10 +22,18 @@ const StyledCellHeader = styled(Table.CellHeader)(
   }
 )
 
+const StyledButtonTableCell = styled(Table.CellHeader)`
+  min-width: 152px;
```
Comment (rafenden) [inline src/dashboard/my-companies/MyCompaniesTable.jsx]: Would be good to see if it looks good on mobile.

Comment (ian-leggett) [reply src/dashboard/my-companies/MyCompaniesTable.jsx]: Unfortunately the table needs work for mobile and I don't feel great about the `min-width: 152px` this was just a fix to stop the button wrapping. The whole home page dashboard definitely needs work for small screens.

Comment (peterhudec) [reply src/dashboard/my-companies/MyCompaniesTable.jsx]: Can't you use `white-space: nowrap`?

File: src/dashboard/my-companies/MyCompaniesTable.jsx [hunk 2]
```diff
@@ -64,6 +73,7 @@ function MyCompaniesTable() {
       <StyledCellHeader>Company name</StyledCellHeader>
       <StyledCellHeader>Last interaction</StyledCellHeader>
       <StyledCellHeader>&nbsp;</StyledCellHeader>
+      <StyledCellHeader>&nbsp;</StyledCellHeader>
```
[the remark this answers is not in the document]
Comment (peterhudec) [reply src/dashboard/my-companies/MyCompaniesTable.jsx]: Also I'm pretty sure that you can remove the `&nbsp`.

File: src/dashboard/my-companies/MyCompaniesTable.jsx [hunk 3]
```diff
@@ -101,6 +111,16 @@ function MyCompaniesTable() {
             latestInteraction.subject
           )}
         </Table.Cell>
+        <StyledButtonTableCell>
+          <Button
+            as={StyledLink}
+            buttonColour={GREY_3}
+            buttonTextColour={TEXT_COLOUR}
+            href={`/companies/${company.id}/interactions/create`}
```
[the remark this answers is not in the document]
Comment (ian-leggett) [reply src/dashboard/my-companies/MyCompaniesTable.jsx]: All done 👍

**Pull request merged**